MANIFEST.in

```diff
@@ -1,4 +1,3 @@
 include LICENSE README.rst CHANGES.rst
-recursive-include axes *.py
```

Comment:
This source code inclusion is already in the stock configuration and thus the definition was duplicate.